packages/x-codemod/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-codemod",
-  "version": "6.9.0",
+  "version": "7.0.0-alpha.0",
```

Comment:
I did include this in the major just to keep it in the same major version range

Reply:
I'm not sure we should be doing it. I don't think that there were any changes made to this package. 🤔 
Unless, we could whip up a v7 `preset-safe` that would include the `components` -> `slots` renaming. 🙈

Reply:
IMO one could argue that packages that are in the same namespace should always follow the major version (pretty much like react does it), but I guess this is a bit personal preference.

Of course: if you do not have changes that make the version change viable this loses a lot of meaning. I get that.

I guess this can go either way.

Thoughts @mui/x ?

Reply:
I'll quickly prepare the v7/preset-safe codemod PR. Sorry for not looking into it sooner. 🙈

Reply:
Not sure how other think about it, I personally feel like releasing every package in the first release of the new major even if there's no diff could be better to set a baseline for further releases.
Having some packages with `v6.*` in major `v7.*` doesn't look great.
That was the reason I didn't rush the initial codemod setup, if we go that line, maybe it can wait. Wdyt @LukasTy?

Reply:
@MBilalShafi that's what I was thinking as well. Even if the package stays at `v7.0.0` throughout the entirety of the major version lifecycle this still makes sense

Reply:
Yeah, I agree with the ideas and proposal. Was just thinking that having at least the basic `codemod` package setup for v0 would be nice. 🙈

Reply:
Resolving it as we introduced the initial `preset-safe`. 👌

Reply:
Thank you @LukasTy for setting that up 🙂